File: Signiture-Backend/kotlin/delivery/store-admin/src/main/java/pcy/study/storeadmin/config/jpa/JpaConfig.java
package pcy.study.storeadmin.config.jpa;

import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EntityScan(basePackages = "pcy.study.db")
@EnableJpaRepositories(basePackages = "pcy.study.db")
public class JpaConfig {
}

File: Signiture-Backend/kotlin/delivery/api/src/main/kotlin/pcy/study/api/config/jpa/JpaConfig.kt
package pcy.study.api.config.jpa

import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;

@Configuration
@EntityScan(basePackages = ["pcy.study.db"])
@EnableJpaRepositories(basePackages = ["pcy.study.db"])
class JpaConfig
